hosts/jenkins-controller/get_secret.py

```diff
@@ -0,0 +1,24 @@
+"""
```

Comment:
Not blocking at this point, but noting down that various checks and linters currently fail due to changes in this PR, e.g.: `reuse lint` `pylint` `inv pre-push`

Reply:
Yes, the in-repo linter and the linter used inside the build framework for `writers.writePython` disagree about style. I think I'll just vendor these few lines into the Nix file that makes the call.

Reply:
Ci is green now, marking this as resolved.